persistent/Database/Persist/Types/Base.hs

```diff
@@ -242,8 +241,6 @@ data PersistException
     deriving (Show, Typeable)
 
 instance Exception PersistException
-instance Error PersistException where
```

Comment:
I'm opening this PR with changed bound version for transformers, so that this can be integrated whenever the new breaking version is released.